### PR TITLE
Feature/deliverable2

### PR DESCRIPTION
**Pull request by rafaelroguz**

### CODE REVIEW

File: src/components/FeaturedProducts/FeaturedProducts.jsx [hunk 1]
```diff
@@ -1,28 +1,65 @@
...
-        return <FeaturedItem itemData={itemData} key={item.id} />;
-      })}
-    </ListContainer>
-  </Container>
-);
+const FeaturedProducts = ({ itemList }) => {
+  const { setLocation } = useLocation();
+  const { isMobile } = useGetScreenSize();
+
+  const handleClickAddToCartButton = (productId) => {
```
Comment (jazzcelaya) [inline src/components/FeaturedProducts/FeaturedProducts.jsx:18]: i see this function is going be used in different parts of the app, so we could perhaps make it a custom hook as I see you're already implemented here. Nice job :)

Comment (jazzcelaya) [inline src/components/FeaturedProducts/FeaturedProducts.jsx:18]: i see this function is going be used in different parts of the app, so we could perhaps make it a custom hook as I see you're already implemented here. We could also just do a helpers folder/file. Nice job :)

**Review by jazzcelaya — APPROVED** [review]
Great work! nice documentation of your process.

File: src/components/FeaturedProducts/FeaturedProducts.jsx (hunk 1 above)
Comment (rafaelroguz) [reply src/components/FeaturedProducts/FeaturedProducts.jsx:18]: I'll do a refactor to handle this on the next deliverable

**Pull request merged**